packages/newspringchurchapp/src/live/LiveContext.js

```diff
@@ -17,6 +17,7 @@ const LiveProvider = (props) => (
 const LiveConsumer = ({ contentId, children }) => (
   <Consumer>
     {(liveStreams) => {
+      if (!contentId) return children(liveStreams);
```

Comment:
Do you feel like this is something that core would accept too? Seems like it would make sense there as well.

Reply:
no, I don't really love the code. the consumer returns either an array or a single item depending on the param. I'm kinda hoping we don't need the banner long term so we can take this out.